### PR TITLE
Init documentation website

### PR DESCRIPTION
**Pull request by ffelten**
Based on Farama template.

I initiated some pages but did not clean everything. I'm wondering how to avoid duplicating the README vs the website.

## Testing
```
pip install . 
cd docs/
pip install -r docs/requirements.txt
make dirhtml _build
```

Open the `docs/build/dirhtml/index.html`  using a browser

### CODE REVIEW

**Comment by ffelten** [comment]
I see this when I click on the side menu: 
image

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#
+html_theme = "furo"
+html_title = "MO-Gymnasium Documentation"
+html_baseurl = "https://mo-gymnasium.farama.org"
+html_copy_source = False
+html_favicon = "_static/img/favicon.png"
+html_theme_options = {
+    # "light_logo": "img/Minari.svg",
+    # "dark_logo": "img/Minari_White.svg",
+    "gtag": "G-R5TRTT6R78",
```
Comment (mgoulao) [inline docs/conf.py:69]: Remove this for now. We need to create one for MO.

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    "versioning": True,
```
Comment (mgoulao) [inline docs/conf.py:70]: Do we want versioning in this project?

File: docs/environments/environments.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+|----------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------|---------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `deep-sea-treasure-v0` <br><img src="https://raw.githubusercontent.com/LucasAlegre/mo-gym/main/screenshots/dst.png" width="200px">                       | Discrete / Discrete                 | `[treasure, time_penalty]`                                    | Agent is a submarine that must collect a treasure while taking into account a time penalty. Treasures values taken from [Yang et al. 2019](https://arxiv.org/pdf/1908.08342.pdf).                                                                                                                          |
```
Comment (mgoulao) [inline docs/environments/environments.md:9]: Update paths to: `https://raw.githubusercontent.com/Farama-Foundation/MO-Gymnasium/main/screenshots/*.png`

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+---
+hide-toc: true
+firstpage:
+lastpage:
+---
+
+[//]: # (```{toctree})
```
Comment (mgoulao) [inline docs/index.md:7]: I think these should be removed

**Review by mgoulao — COMMENTED** [review]
Building:
```
make dirhtml
python -m http.server  --directory _build
```
OR:
```
sphinx-autobuild -b dirhtml . _build
```

**Pull request merged**